{{cookiecutter.project_slug}}/compose/production/django/entrypoint

```diff
@@ -5,9 +5,6 @@ set -o pipefail
 set -o nounset
 
 
-# N.B. If only .env files supported variable expansion...
-export CELERY_BROKER_URL="${REDIS_URL}"
```

Comment:
I'm strongly -1 here: this line is here for exactly the reason mentioned in the comment, to avoid configuration duplication.

Reply:
@webyneter Thank you for the feedback. I understand your point; I was reluctant to change it initially for this reason. However, here are some comments:
- The `CELERY_BROKER_URL` is always defined, even though there are cases (like the one that this PR is based on) which Celery is not used. Of course, this doesn't break anything, but why define it if it is not used anywhere?
- Since it is a configuration variable, it would be better to be defined where the rest of the configuration variables are defined, i.e. in the `.django` file
- Even though very unlikely, there could be a case when Celery uses a different Redis server than the one used by the Django cache

For the above reasons, in my opinion, it is better to have the `CELERY_BROKER_URL` defined in the `.django` file. Of course I could be completely wrong :smile: :flushed: so I would like to hear your thoughts.

Reply:
> The CELERY_BROKER_URL is always defined, even though there are cases (like the one that this PR is based on) which Celery is not used. Of course, this doesn't break anything, but why define it if it is not used anywhere?

Sure, go ahead and surround `CELERY_BROKER_URL` with Jinja2 if-clause.

> Since it is a configuration variable, it would be better to be defined where the rest of the configuration variables are defined, i.e. in the .django file

Not much we can do here as`.env` files don't support variable expansion as both of us know.

> Even though very unlikely, there could be a case when Celery uses a different Redis server than the one used by the Django cache

Yes, that's exactly what I was about to address in one of the upcoming PRs -- good for your to have spotted this! Indeed, having `DJANGO_CACHE_URL=redis://redis:6379/0` and `CELERY_BROKER_URL=redis://redis:6379/1` is the best we could do. Tell you what, @demestav, how about you send us a PR? :)

Reply:
> Sure, go ahead and surround CELERY_BROKER_URL with Jinja2 if-clause.

@webyneter please check the new commit. I hope that resolves it.

> Tell you what, ..., how about you send us a PR? :)

I will do that!

A general question about the PR process. New changes are constantly made to the upstream repository and as a result conflicts occur on the forked repository. What is the process to follow in such occasions? Do I constantly rebase the fork? Thanks.

Reply:
> Do I constantly rebase the fork?

I'm afraid, yes, rebasing or merging is the only way I could think of and I've given these thoughts quite some time. Unless, of course, I'm missing something.

Reply:
As long as you keep up with upstream changes regularly there shouldn't be a lot of merge conflicts to resolve.